### ISSUE TITLE
How do I edit the social login buttons (UserAccounts - Semantic -UI)

### ISSUE THREAD

**Issue by mervynteo** [meteor-useraccounts/core#366]
For social button edits, I installed the `aldeed:template-extension`. 

In config.js, I added: `Template.customAtSocial.replaces("atSocial");`. 
However theres a TypeError, says "cannot call method 'replaces' of 'undefined'?

This is currently the router. Removing the `template: ""` line does not work. Error still exists. 

<pre lang="javascript"><code>
AccountsTemplates.configureRoute('signIn', {  //login
  name: 'login',
  path: '/login',
  //template: 'atTemplate',
  layoutTemplate: 'atLayout',
  redirect: '/'
});

AccountsTemplates.configureRoute('signUp', {  //registration
  name: 'register',
  path: '/register',
  //template: 'atTemplate',
  layoutTemplate: 'atLayout',
  redirect: '/'
});
</code></pre>

I want the social login as logo buttons without any words, lined up in a row above the email/ password login. Also looking to change the size of the buttons as well. 

I understand I need to tweak the config.js code (see above) which is the existing code within the semantic-UI file. How do I then disable the existing social login buttons? (I already have `Template.customAtSocial.replaces("atSocial");` started, assuming its working): 

// This is the 2nd part of code in config.js 

`<template name="customAtSocial">
  <button class="at-social-btn ui labeled icon {{disabled}} {{serviceName}} large button" id="at {{name}}" name="{{name}}">
     <i class="{{iconClass}} icon"></i>.....`

Which parts should I edit?

**Comment by splendido** [comment]
To override default social buttons please run from the console

```shell
meteor add aldeed:template-extension
```

then, under the `client` folder of your app create a `my_at_social.html` file including:

```html
<template name="myAtSocial">
  <button class="at-social-btn ui big icon {{disabled}} {{serviceName}} button" id="at-{{name}}" name="{{name}}">
    <i class="{{iconClass}} icon"></i>
  </button>
</template>
```

and a `my_at_social.js` file with:

```javascript
Template.myAtSocial.replaces("atSocial");
```


That's it ;-)

![squaredsocialbutton](https://cloud.githubusercontent.com/assets/6148980/7337482/ac5a4402-ec2b-11e4-8a70-e59ad8369e41.png)

**Comment by mervynteo** [comment]
@splendido It finally worked!! Thanks a million and I love your project :)

**Comment by splendido** [comment]
You're welcome! 

Tnx

**Comment by ryandeussing** [comment]
:+1: thumbs-up

**Comment by kynwu** [comment]
Can I simply modify "Sign Up With Facebook" text?
cause the translation from t9n wasn't complete (only translate "With")
and has some grammar issue...

**Comment by splendido** [comment]
you could try to contribute to `softwarerero:accounts-t9n` to fix the translation problem.
But in case that's not an easy fix I'd say we'll have to wait for v2.0 where the text configuration will be way more comprehensive
